common/src/View/MapView2D.cpp

```diff
@@ -201,7 +203,7 @@ namespace TrenchBroom {
             const std::vector<Model::Brush*>& selectionBrushes = document->selectedNodes().brushes();
             assert(!selectionBrushes.empty());
 
-            const Model::BrushBuilder brushBuilder(document->world(), worldBounds);
+            const Model::BrushBuilder brushBuilder(document->world(), worldBounds, game->defaultFaceAttribs());
```

Comment:
For this brush, you don't need to apply the defaults at all, it will be removed immediately.